addons/project_todo/static/tests/tours/project_todo_main_functions.js

```diff
@@ -75,6 +75,12 @@ registry.category("web_tour.tours").add('project_todo_main_functions', {
     trigger: ".o_project_task_kanban_view",
 },
 {
+    trigger: ".oe_kanban_content",
+    run: () => document.querySelector(".btn.o-no-caret").click(),
```

Comment:
```suggestion
    trigger: ".oe_kanban_content .btn.o-no-caret",
    run: "click",
```
Cannot we do that? 🧐

Reply:
Ah, indeed it looks possible